test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,272 @@
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await unlockWallet(driver);
+
+        // on testdapp, add created tokens in wallet
+        await openDapp(driver, contractAddress);
+        await driver.findClickableElement('#deployButton');
+        await driver.clickElement({
+          text: 'Add Token(s) to Wallet',
+          tag: 'button',
+        });
+        await switchToNotificationWindow(driver);
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+
+        // transfer token from dapp
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+        await driver.clickElement({ text: 'Transfer Tokens', tag: 'button' });
+        await switchToNotificationWindow(driver);
+        await driver.waitForSelector({ text: '1.5 TST', tag: 'h1' });
+
+        // edit gas fee
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+        await driver.clickElement(
+          { text: 'Edit suggested gas fee', tag: 'button' },
+          10000,
+        );
+        await editGasfeeCustomToken(driver);
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        // check that transaction has completed correctly and is displayed in the activity list
+        await driver.switchToWindowWithTitle(
+          WINDOW_TITLES.ExtensionInFullScreenView,
+        );
+        await driver.clickElement({ tag: 'button', text: 'Activity' });
+        await driver.waitForSelector({
+          css: '[data-testid="transaction-list-item-primary-currency"]',
+          text: '-1.5 TST',
+        });
+
+        // check token amount is correct after transaction
+        await driver.clickElement({
+          text: 'Tokens',
+          tag: 'button',
+        });
+        const tokenAmount = await driver.findElement(
+          {
+            css: '[data-testid="multichain-token-list-item-value"]',
+            text: '8.5 TST',
+          },
+          { timeout: 10000 },
+        );
+        assert.ok(tokenAmount, 'Token amount is not correct');
+      },
+    );
+  });
+
+  it('transfer custom tokens from dapp without specifying gas', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        smartContract,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await unlockWallet(driver);
+
+        // on testdapp, add created tokens in wallet
```

Comment:
same as above. Adding  the token in the state with fixtures would allow us to skip this step

Reply:
Done, thanks !